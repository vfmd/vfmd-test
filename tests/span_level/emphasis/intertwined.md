one *two _three* four_ five

one **two _three** four_ five

one *two **three* four** five

one **two *three** four* five

one _two *three_ four* five

one __two *three__ four* five

one _two __three_ four__ five

one __two _three__ four_ five
