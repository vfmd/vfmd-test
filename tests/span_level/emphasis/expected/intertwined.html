<p>one <em>two _three</em> four_ five</p>

<p>one <strong>two _three</strong> four_ five</p>

<p>one <em>two <em><em>three</em> four</em></em> five</p>

<p>one <em><em>two <em>three</em></em> four</em> five</p>

<p>one <em>two *three</em> four* five</p>

<p>one <strong>two *three</strong> four* five</p>

<p>one <em>two <em><em>three</em> four</em></em> five</p>

<p>one <em><em>two <em>three</em></em> four</em> five</p>